### PR TITLE
add origamijs@0.5.1 and update .gitignore

### LINKED ISSUE #9060: [Request] origamijs
**Library name:** Origamijs
**Git repository url:** http://github.com/raphamorim/origami.js
**npm package url(optional):**  https://www.npmjs.com/package/origamijs
**License(s):** GPL-3.0
**Official homepage:** http://origamijs.com
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #9060 
@x09326 please help me review this pull request, thank you.
add .gitignore to ignore v0.0.2 because v0.0.2 doesn't exist on npm and has no origami.js on git repo, author said he want to forget v0.0.2
https://github.com/raphamorim/origami.js/issues/39#issuecomment-249425016
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/raphamorim/origami.js
 * Official website (optional, not the repository): http://origamijs.com
 * NPM package url (optional): https://www.npmjs.com/package/origamijs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 568
   - Count of watchers: 25
   - Count of forks: 34
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 235
   - Downloads in the last month: 365

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

npm auto-update config added and add .gitignore to ignore v0.0.2 because
author don't care, close #9060, cc @raphamorim

### CODE REVIEW

File: ajax/libs/origamijs/0.5.1/origami.js [hunk 1]
```diff
@@ -0,0 +1,1153 @@
+/*!
+ * Origami.js 0.5.0
```
Comment (x09326) [inline ajax/libs/origamijs/0.5.1/origami.js]: @kennynaoh 
Here is 0.5.0.
Would you confirm this?

**Comment by kennynaoh** [comment]
@x09326 
That is just a typo (https://github.com/raphamorim/origami.js/issues/39)
And removed the blank line.
Please help me review it again.
Thanks!

**Comment by raphamorim** [comment]
Make sense 👍

**Comment by kennynaoh** [comment]
:bowtie:

**Comment by raphamorim** [comment]
Ref: https://github.com/raphamorim/origami.js/issues/42

**Comment by kennynaoh** [comment]
Thanks @raphamorim :sparkles:

**Comment by kennynaoh** [comment]
@x09326 
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
"npm auto-update config added and add .gitignore to ignore v0.0.2 because **author don't care**"
I didn't get what do you mean, the author doesn't care for what?

**Comment by kennynaoh** [comment]
The author said let's forget `v0.0.2`

**Comment by PeterDaveHello** [comment]
1. Where did the author say that? Give us the reference in PR comment.
2. Clarify your comment and commit message.

**Comment by kennynaoh** [comment]
@x09326 
I have clarified the comment in commit message and add ref in PR comment
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**